README.md

```diff
@@ -22,20 +22,53 @@ yarn add @flatfile/api
 [![Try it out](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/edit/flatfile-typescript-sdk-7xtsm5?file=app.ts&view=editor)
```

Comment:
This Stackblitz code was already different than our previous usage example and it includes creating a workbook, so I don't think it needs to be updated.